Content.Client/_Corvax/Barks/BarkSystem.cs

```diff
@@ -0,0 +1,95 @@
+// SPDX-FileCopyrightText: 2025 GoobBot <uristmchands@proton.me>
+// SPDX-FileCopyrightText: 2025 Zekins <zekins3366@gmail.com>
+//
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+using Content.Client.Audio;
+using Content.Shared._Corvax.Speech.Synthesis;
+using Content.Shared.Chat;
+using Content.Shared.Corvax.CorvaxVars;
+using Robust.Client.Audio;
+using Robust.Client.Player;
+using Robust.Client.ResourceManagement;
+using Robust.Shared.Audio;
+using Robust.Shared.Audio.Systems;
+using Robust.Shared.Configuration;
+using Robust.Shared.Timing;
+using Robust.Shared.Utility;
+
+namespace Content.Client._Corvax.Speech.Synthesis.System;
+
+/// <summary>
+/// The system responsible for the sound transmission for each subscriber
+/// </summary>
+public sealed class BarkSystem : EntitySystem
+{
+    [Dependency] private readonly AudioSystem _audio = default!;
+    [Dependency] private readonly IConfigurationManager _cfg = default!;
+    [Dependency] private readonly IEntityManager _entityManager = default!;
+    [Dependency] private readonly IPlayerManager _player = default!;
+
+    private const float MinimalVolume = -10f;
+    private const float WhisperFade = 4f;
+
+    public override void Initialize()
+    {
+        SubscribeNetworkEvent<PlayBarkEvent>(OnPlayBark);
+    }
+
+    public void RequestPreviewBark(string barkVoiceId)
+    {
+        RaiseNetworkEvent(new RequestPreviewBarkEvent(barkVoiceId));
+    }
+
+    private void OnPlayBark(PlayBarkEvent ev)
+    {
+        var sourceEntity = _entityManager.GetEntity(ev.SourceUid);
+        if (!_entityManager.EntityExists(sourceEntity) || _entityManager.Deleted(sourceEntity) || !HasComp<TransformComponent>(sourceEntity))
+            return;
+
+        if (_player.LocalEntity != null && HasComp<TransformComponent>(_player.LocalEntity.Value))
+        {
+            var sourceTransform = Transform(sourceEntity);
+            var playerTransform = Transform(_player.LocalEntity.Value);
+
+            if (sourceTransform.Coordinates.TryDistance(EntityManager, playerTransform.Coordinates, out var distance) &&
+                distance > SharedChatSystem.VoiceRange)
```

Comment:
Keep the logic operators at the start of each line so its easier to read through. Especially with long lines of code preceding them, tends to go offscreen on smaller displays.
```suggestion
            if (sourceTransform.Coordinates.TryDistance(EntityManager, playerTransform.Coordinates, out var distance)
                && distance > SharedChatSystem.VoiceRange)